src/configs/all.ts

```diff
@@ -193,6 +193,7 @@ export const rules = {
     "match-default-export-name": true,
     "new-parens": true,
     "newline-before-return": true,
+    // "newline-per-chained-call": Errors
```

Comment:
I don't know if we should exclude it. We should rather find a reasonable default value.

From the "Errors" I can deduce that there are many lint errors within this project if you enable the rule. In this case you can just disable it in `tslint.json`